src/disk.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import psutil
+import os
+import sys
+from psutil._common import bytes2human
+
+
+def _get_default_path():
+    """Get the default path for the current platform"""
+    if sys.platform == "win32":
+        return "C:"
+    elif sys.platform == "darwin":
+        return "/System/Volumes/Data"
+    elif (
+        # IF kind of linux AND WSL is installed
+        sys.platform.startswith("linux") and os.path.exists("/usr/lib/wsl")
+    ):
+        return "/usr/lib/wsl/drivers"
+    else:
+        return "/"
+
+
+def get_disk_usage_percent(path=None):
+    """Display disk usage as a percentage"""
+    if path is None:
+        path = _get_default_path()
```

Comment:
This snippet is now duplicated for each function. It is perhaps more readable but also a bit repetitive, maybe a reasonable trade-off at this point in the project